packages/gatsby/src/utils/get-ssl-cert.js

```diff
@@ -15,9 +15,12 @@ module.exports = async ({ name, certFile, keyFile, directory }) => {
   // check that cert file and key file are both true or both false, if they are both
   // false, it defaults to the automatic ssl
   if (certFile ? !keyFile : keyFile) {
-    report.panic(
-      `for custom ssl --https, --cert-file, and --key-file must be used together`
-    )
+    report.panic({
+      id: `11521`,
+      context: {
```

Comment:
Is this bit necesary? maybe empty object or just remove it as we're not using it in the message.

Wdyt?

Reply:
I had tried without passing context, but apparently the message is needed for the gatsby-telemetry package.  I couldn't solve it any other way.  If you have any idea how to overcome this, I can try to solve it.  Thanks.

Reply:
Let's use empty `context: {}` - this should work ok. But this is separate issue, and we should fix this in telemetry package to not crash on undefined context